--- fbzx-2.5.0/fbzx.desktop~	2011-04-07 00:34:20.000000000 +0200
+++ fbzx-2.5.0/fbzx.desktop	2011-04-13 10:12:09.159431293 +0200
@@ -1,12 +1,14 @@
 [Desktop Entry]
 X-MultipleArgs=false
 Type=Application
-Version=2.4.3
 Encoding=UTF-8
 Name=FBZX
 GenericName=A Sinclair ZX Spectrum emulator
+GenericName[pl]=Emulator Sinclair ZX Spectrum
 Comment=Play with the old, great games of the venerable ZX Spectrum
+Comment[pl]=Zagraj w stare gry na ZX Spectrum
 TryExec=fbzx
 Exec=fbzx
-Categories=Game;
+Categories=Utility;Emulator
 Icon=fbzx
+# vi: encoding=utf-8
